build.rs

```diff
@@ -0,0 +1,19 @@
+use std::process::Command;
+use std::env::remove_var;
+
+fn main() {
+    // Remove sysroot flag... well remove all of them for now.
+    remove_var("RUSTFLAGS");
```

Comment:
What does this do?

Reply:
Xargo sets `RUSTFLAGS="--sysroot <some_path>"`, which causes rustc to not be able to find `core` for wasm32-unknown-unknown. This undoes that.

Reply:
And that doesn't interfere with building the kernel?

Reply:
No, it's removed for things underneath the buildscript (which is only `userspace`)